app/controllers/task_controller.rb

```diff
@@ -30,12 +30,45 @@ def delete
   	redirect_to action: "index"
   end
 
+  def edit
+  	@edit_id = params[:edit_id]
```

Comment:
This applies to a few other instance variables that you use throughout these action methods as well